.github/workflows/ci.yml

```diff
@@ -192,17 +191,31 @@ jobs:
           fetch-depth: 2147483647
       - run: sudo apt-get update && sudo apt-get install -y kcov
       - run: ./zig/download.sh
+
+      # Dummy devhub run - checks that all the devhub tests pass in CI. They are run again, in main
+      # once merged. Kcov is skipped to avoid adding to the pipeline time.
+      - run: sudo -E ./zig/zig build scripts -- devhub --sha=${{ github.sha }} --skip-kcov
```

Comment:
It would be nice to not need both the `--skip-kcov` and the `--devhub` flag to release, but without them, devhub takes ~20 minutes in CI which would add to the total pipeline time...